src/org/usfirst/frc/team2706/robot/commands/EjectCubeWithIR.java

```diff
@@ -0,0 +1,43 @@
+package org.usfirst.frc.team2706.robot.commands;
+
+import org.usfirst.frc.team2706.robot.Robot;
+import org.usfirst.frc.team2706.robot.subsystems.Intake;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class EjectCubeWithIR extends Command {
+
+    private Intake exhale;
+    
+    /**
+     * Allows us to use the methods in 'Intake'
+     */
+    public EjectCubeWithIR() {
+        exhale = Robot.intake;
+        this.requires(Robot.intake);
+    }
+    
+    /**
+     * Turns on the motors on to eject the cube
+     */
+    public void execute() {
+        exhale.exhaleCube(1.0); //TODO check out if correct
```

Comment:
The intake shouldn't be running at full speed because it will destroy the intake wheels